server/container_create.go

```diff
@@ -850,6 +850,15 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 		specgen.AddBindMount("/etc/hosts", "/etc/hosts", options)
 	}
 
+	// bind mount user defined secrets
+	mounts, err := secretMount(mountLabel)
+	if err != nil {
+		logrus.Warn("error getting secret mounts")
```

Comment:
Should probably return an error.

Reply:
okay, I wasn't sure if we wanted create container to fail if getting the secret mounts failed.

Reply:
No if users are running on RHEL and this fails, it could lead to other issues.  Better to fail,  Hopefully it will never happen, only if there is a configuration issue.

Reply:
fixed.